[DOC] Create Kafka Bridge quick start (#1940)
strimzi/strimzi-kafka-operator: documentation/book/assembly-kafka-bridge-quickstart.adoc

// This assembly is included in the following assemblies:
//
// assembly-using-the-kafka-bridge.adoc

// Save the context of the assembly that is including this one.
// This is necessary for including assemblies in assemblies.
// See also the complementary step on the last line of this file.
:parent-context-using-the-kafka-bridge: {context}
:context: {parent-context-using-the-kafka-bridge}-bridge-quickstart

[id='assembly-kafka-bridge-quickstart-{context}']
= Kafka Bridge quick start

> Use this quick start to try out the {ProductName} Kafka Bridge in your local development environment. You will learn how to:

Reviewer: ```suggestion
Use this quickstart to try out the {ProductName} Kafka Bridge in your local development environment. You will learn how to:
```

> * Deploy the Kafka Bridge to your OpenShift cluster

Reviewer 2: ```suggestion
* Deploy the Kafka Bridge to your Kubernetes cluster
```

> * Expose the Kafka Bridge Service to your local machine

Reviewer 3: We use lower case 'service' in the main guide

* Produce messages to topics, including to a specific partition
* Create a Kafka Bridge consumer for use in this quick start

> * Perform basic consumer operations

Reviewer 4: is "for use in this quick start" needed? It's clear that we are going to use the consumer for the quick start, or?
I guess that this point could be merged with the last one as:

* Create a Kafka Bridge consumer and perform basic consumer operations

Author: Good suggestion, thank you.

> HTTP requests are formatted as curl commands that you can copy and paste to your terminal. You can use an API testing tool to call the API if preferred. As described in xref:con-requests-kafka-bridge-str[Requests to the {ProductName} Kafka Bridge], all POST requests used in this quick start supply the following Content-Type header:

Reviewer 4: what API testing tool are you referring to? Maybe a curl replacement with a UI like Postman?

Author: Yes, something like Postman. Perhaps I should remove this sentence.

Author: Removed the sentence on API testing tools.

[source,http,subs=attributes+]
----
Content-Type: application/vnd.kafka.v2+json

> ----

Reviewer 4: it depends ... if you are describing how to do POST for sending messages and you are using JSON as payload in your examples, then the content type is `application/vnd.kafka.json.v2+json`

Author: Excellent point. I'll update this content and ask you to re-check it.

Also, I think "Creating a Kafka Bridge consumer" should explain _why_ we are using a Content-Type header of `application/vnd.kafka.v2+json`.

Author: Please check the updated information when you get a moment, thank you.

> You must have access to a Kubernetes cluster. To run and manage a Kubernetes cluster locally, use a tool such as Minikube, CodeReady Containers, or MiniShift.

Reviewer 3: How will this equate downstream?

.Prerequisites for the quick start

* Cluster administrator access to a local or remote Kubernetes cluster. 
* {ProductName} is installed.
* A running Kafka cluster, deployed by the Cluster Operator, in a Kubernetes namespace.

> * The Cluster Operator `RoleBindings` are installed.

Reviewer 2: Isn't this point kind of already expected when we already before say we expect _Strimzi to be installed_ and _Kafka cluster to be deployed_? It seems strage to mention it like this because it migth suggest this is something the user has to do extra.

Reviewer 4: +1

Author: I think I got confused after creating my own cluster. I'll remove this prereq.

> * The Entity Operator is deployed and running.  

Reviewer 2: We should try to avoid confusion that this needs to be deployed separately.

```suggestion
* The Entity Operator is deployed and running as part of the Kafka cluster.  
```
